doc/fulcrum-example-config.conf

```diff
@@ -531,7 +531,7 @@ rpcpassword = hunter1
 # default in your torrc, then specify the settings used here so that Fulcrum
 # may use the Tor proxy to discover .onion peers.
 #
-#tor_proxy = 9050   # e.g. localhost 9050. IP addr or hostname ok too: 10.0.0.1:9150, fooproxy.com:9050, etc.
+#tor_proxy = 9050   # e.g. localhost:9050. IP addr or hostname ok too: 10.0.0.1:9150, fooproxy.com:9050, etc.
```

Comment:
Should it be `#tor_proxy = localhost:9050 # IP addr or hostname ok too: 10.0.0.1:9150, fooproxy.com:9050, etc.`?

Reply:
I mean you can just specify a port.. or both host:port.. if no host, defaults to localhost... is this confusing? Then maybe it should be clarified. 99.9999% of users would want to just specify a port since it makes not that much sense to run a proxy on another host (although i can imagine some setups that have this).

Reply:
Oh wait I'm fucking retarded man. I dunno what's wrong with me lately. Yes, your diff is correct. I misread your diff backwards like you wanted to **remove** the colon!!  Yeah the colon is good.

Reply:
> is this confusing?

The missing colon was confusing only when I started thinking it could have been intentional.  If one can specify only a port, can one specify only an address?  In the scenario where one could specify only an address, it is feasible that the intention was to have localhost and 9050 as separate example options (perhaps with comma space between them instead of just space).  If port is required but address is optional, then the merged PR is correct.

Reply:
Yep. Port req'd, addr optional.